js/copy-service/Parser/Parser.js

```diff
@@ -328,6 +328,20 @@ class Parser {
 
   /* eslint-disable brace-style */
 
+  /**
+   *
+   * @param {*} key
+   * @returns
+   */
+  static _getParentKey(key) {
```

Comment:
```
_getRelativeKey(partialKey, parentKey) {
  if (!_.startsWith(partialKey, '.') {
     return partialKey; // not relative
  }

  const prefixRegex = /^\.*/;
  const prefix = partialKey.match(prefixRegex)[0];
  const parentSteps = prefix.length;
  const withoutPrefix = partialKey.replace(prefixRegex, '');

  const parentPortion = parentKey.split('.').slice(0, -1 * parentSteps).join('.');
  return `${parentPortion}.${withoutPrefix};
}
```

This should support multi-level keys like `....example.key`.

Reply:
oh! I totally did not realize you had written this before committing 9a469d0. Let me revise...